markdown/docs/tools/cli/installation.md

```diff
@@ -45,29 +45,61 @@ To refresh the variables:
  source ~/.zshrc    # For zsh
 ```
 
-## Docker
+# Docker
 
-Install [Docker](https://docs.docker.com/get-docker/) first, then use docker to build the image using the following command :
-``` 
-docker build -t asyncapi/cli:latest . 
-``` 
-and run the image using the following command :
+Install [Docker](https://docs.docker.com/get-docker/) first, then you can either **build the image locally** or **pull it from Docker Hub**.
+
+## 🏗️ Option 1: Build the image locally
+
+Build the image using the following command:
+
+```bash
+docker build -t asyncapi/cli:latest .
+```
+
+Then, run the image using:
+
+```bash
+docker run --rm -it --user=root -v [ASYNCAPI SPEC FILE LOCATION]:/app/asyncapi.yml -v [GENERATED FILES LOCATION]:/app/output asyncapi/cli [COMMAND HERE]
```

Comment:
@akshat99812 I would recommend keeping the original formatting for command syntax and example where each parameter is on a new line. This gives you a better readability and shorter line length.